src/frontend/web_application/src/components/form/TextFieldGroup/style.scss

```diff
@@ -1,11 +1,24 @@
 @import '../../../styles/common';
 
 .m-text-field-group {
+  position: relative;
+
   &__label {
     display: block;
   }
 
   &__errors {
-    margin-top: map-get($co-form__spacing, 'medium');
+    position: relative;
+    margin-top: -(map-get($co-form__spacing, 'medium'));
```

Comment:
hmm there's a problem here: we shouldn't have to remove a padding.

I suppose it's my mistake, I did say a component should not have margin because it's more difficult to remove than add it. But I didn't do it there.

So it's the same remark as previous. the padding is fixed here but not for other form components.

I propose to not fix this margin here. Merge this PR. Then create a new one that fix the spacing for all cases